go/api/command/command.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.22.0
+// 	protoc-gen-go v1.25.0
```

Comment:
It is better to have separate PR for this?
Or this is necessary for this PR?

Reply:
https://github.com/bazelbuild/remote-apis-sdks/pull/219